### PR TITLE
Gantry clean start

### PR DESCRIPTION
**Pull request by autodidaddict**
* Updated readmes so building and running all gantry components should be easy now
* Removed the hard-coded actor public keys
* Removed the use of a host manifest for smoother `gantry-server` start  - should now work regardless of the public keys of the actors.
* Also fixed a bug in `streams` where it would break if the root container didn't exist. This actor now asks the blobstore provider to create the `gantry` container upon startup

@Spora-GH - This should hopefully fix issues you've had running Gantry. Make sure you also grab the latest `wascc-host` repo as I've updated the fs-provider asset.

### CODE REVIEW

**Comment by autodidaddict** [comment]
This PR closes #4

File: catalog/src/lib.rs [hunk 1]
```diff
@@ -29,17 +29,18 @@ lazy_static! {
...
 }
 
-actor_handlers! { codec::messaging::OP_DELIVER_MESSAGE => handle_message,
-codec::core::OP_BIND_ACTOR => handle_config,
-codec::core::OP_HEALTH_REQUEST => health }
+actor_handlers! {
+    codec::messaging::OP_DELIVER_MESSAGE => handle_message,
+    codec::core::OP_BIND_ACTOR => handle_config,
+    codec::core::OP_HEALTH_REQUEST => health
+}
```
Comment (brooksmtownsend) [inline catalog/src/lib.rs:36]: I wasn't going to say anything before, but I really do like this formatting better than having things on the same lines as { }

File: server/src/main.rs [hunk 2]
```diff
@@ -37,21 +45,82 @@ fn handle_command(cmd: CliCommand) -> Result<(), Box<dyn ::std::error::Error>> {
...
+    )?;
+
+    // -- per-actor configuration
+    // Note that these actors might attempt to use capabilities in response to being configured
+    // so their capability providers must be bound _before_ the following configuration takes place
+
+    // Provide the root operator and signing keys to the catalog
+    host.bind_actor(
+        &catalog_id,
+        &catalog_id,
```
Comment (brooksmtownsend) [inline server/src/main.rs:102]: Does this just allow the catalog actor to interface with itself? Same comment for a few lines below for the streams actor

File: streams/README.md [hunk 3]
```diff
@@ -4,4 +4,36 @@ The Gantry server requires a number of actor modules to comprise the bulk of its
...
+║ K/V Store                                                                  ║
+║ Messaging                                                                  ║
+║ Logging                                                                    ║
+╠════════════════════════════════════════════════════════════════════════════╣
+║                                    Tags                                    ║
+╠════════════════════════════════════════════════════════════════════════════╣
+║ None                                                                       ║
+╚════════════════════════════════════════════════════════════════════════════╝
+```
+If you've built the catalog module correctly, you'll see its 3 capabilityes, and the `Module` field (the one that starts with **M** is the unique ID/subject/public key of the catalog actor that you need to use for the `CATALOG_ACTOR` environment variable when you use `make` to build the streams actor.
```
Comment (brooksmtownsend) [inline streams/README.md:39]: `capabilityes` typo. I hate being this nitpicky

File: catalog/src/lib.rs (hunk 1 above)
Comment (autodidaddict) [reply catalog/src/lib.rs:36]: Yeah this is much nicer, and `cargo fmt` actually leaves this intact.

File: server/src/main.rs (hunk 2 above)
Comment (autodidaddict) [reply server/src/main.rs:102]: It lets me send configuration values to the catalog actor. There's a special clause in waSCC host that allows an actor to talk to itself without a custom capability in the token, so this is the convention I've been using to send values into actors directly.

File: streams/README.md (hunk 3 above)
Comment (autodidaddict) [reply streams/README.md:39]: /giphy tableflip

File: server/src/main.rs (hunk 2 above)
Comment (brooksmtownsend) [reply server/src/main.rs:102]: Okay I figured actors would be allowed to interface internally, makes sense.

**Pull request merged**